templates/todo/api/js/package.json

```diff
@@ -24,6 +24,7 @@
     ]
   },
   "dependencies": {
+    "@azure/app-configuration-provider": "1.0.0-preview.4",
```

Comment:
any stable version?

Reply:
No. latest still in preview https://github.com/Azure/AppConfiguration-JavaScriptProvider/tags